packages/itwinui-react/src/core/Footer/Footer.tsx

```diff
@@ -146,7 +147,15 @@ export const Footer = Object.assign(
                   {index > 0 && <FooterSeparator />}
                   <FooterItem>
                     {element.url ? (
-                      <a href={element.url} target='_blank' rel='noreferrer'>
+                      <a
+                        href={element.url}
+                        target='_blank'
+                        rel='noreferrer'
+                        className={'iui-anchor'
```

Comment:
You don't need to go through such great lengths to add the class here. Using the `Anchor` component will suffice.

Reply:
Will do.